pkg/util/decimal.go

```diff
@@ -0,0 +1,67 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+)
+
+const maxAllowedPrecision = 8
```

Comment:
Enough for GAS, but too low for NeoFS, for example.

Reply:
Fixed.